src/rules/no-import-module-exports.js

```diff
@@ -0,0 +1,72 @@
+import minimatch from 'minimatch'
+import path from 'path'
+import readPkgUp from 'read-pkg-up'
+
+function getEntryPoint(context) {
+  try {
```

Comment:
C it is. Here's the [diff](https://github.com/benmosher/eslint-plugin-import/pull/804/commits/e2a8ccb337a373e71823dcdaec3f76fdc085f7ae).

Those using `jsnext:main` can always add their second entry to the exceptions in the rule options. I will add something to that effect in the docs.